src/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.fs

```diff
@@ -312,7 +312,19 @@ type FSharpDependencyManager(outputDirectory: string option, useResultsCache: bo
         //   if a path was supplied if it was rooted then use the rooted path as the root
         //   if the path wasn't supplied or not rooted use the temp directory as the root.
         let specialDir =
-            Path.Combine(Environment.GetFolderPath(System.Environment.SpecialFolder.UserProfile), ".packagemanagement", "nuget")
+            let getProfilePath =
+                // If it has a directory seperator remove it
+                let path = Environment.GetFolderPath(System.Environment.SpecialFolder.UserProfile)
+
+                if
+                    (path.EndsWith(Path.DirectorySeparatorChar.ToString()))
+                    && not (path.EndsWith(Path.AltDirectorySeparatorChar.ToString()))
```

Comment:
```suggestion
                    ||  (path.EndsWith(Path.AltDirectorySeparatorChar.ToString()))
```

Why DirectorySeparatorChar && NOT AltDirectorySeparatorChar ?
I would say we should not duplicate the separator if it is either of the two.